src/inverted_index/inverted_index.h

```diff
@@ -10,7 +10,7 @@
 #define __INVERTED_INDEX_H__
 
 #include "redisearch.h"
-#include "buffer.h"
+#include "buffer/buffer.h"
```

Comment:
Remember to remove this change from all the C files (except for the test files where it is not possible)

Reply:
I tried actually but iirc that was breaking CI.

But let me double check.